doc/components.md

```diff
@@ -199,7 +199,7 @@ The simplest `ParallaxComponent` is created like this:
 ```dart
 @override
 Future<void> onLoad() async {
-  final parallaxComponent = await loadParallaxComponent(['bg.png', 'trees.png']);
+  final parallaxComponent = await loadParallaxComponent(['bg.png', 'trees.png'], Vector2.all(100));
```

Comment:
Shouldn't the example usage use the size of the game, I think that is how it most often will be used?

Reply:
Or put the original comment back?

Reply:
Fixed